### PR TITLE
Dropping black and use ruff-format

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

The Ruff formatter is an extremely fast Python code formatter designed as a drop-in replacement for [Black](https://pypi.org/project/black/), available as part of the ruff CLI via ruff format.

### Details

- https://docs.astral.sh/ruff/settings/#format_docstring-code-format

### CODE REVIEW

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

File: pyproject.toml [hunk 1]
```diff
@@ -126,7 +113,7 @@ quiet-level = 3
 omit = [
     'pyvista/ext/coverage.py',
     'pyvista/conftest.py',
-    # kept for backwards compatibility:
+    # uept for backwards compatibility:
```
Comment (tkoyama010) [inline pyproject.toml:116]:
```suggestion
    # kept for backwards compatibility:
```

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Comment by banesullivan** [comment]
Looks like this has some conflicts @tkoyama010?

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Review by banesullivan — APPROVED** [review]

**Pull request merged**